st2actions/st2actions/workflows/workflows.py

```diff
@@ -40,10 +47,15 @@
     queues.WORKFLOW_ACTION_EXECUTION_UPDATE_QUEUE,
 ]
 
+WORKFLOW_ENGINE = "workflow_engine"
```

Comment:
There is already a definition at https://github.com/StackStorm/st2/blob/master/st2actions/st2actions/cmd/workflow_engine.py#L40. Can we keep this here and modify workflow_engine.py to reuse this constant?

Reply:
Addressed https://github.com/StackStorm/st2/pull/5463/files#diff-8570f80aaad5c1850cb5d35d826b860901e4d95b97b3db631c486fb6ad6901e1R40